pandablocks/ioc/ioc.py

```diff
@@ -84,6 +91,64 @@ def _when_finished(task):
     create_softioc_task = None
 
 
+async def _handle_file(request: web.Request) -> web.Response:
+    """Handles HTTP GET requests for individual bob files.
+
+    This function will handle incoming requests for .bob files. And send back
+    a HTTP response with the bob file in question or 404. To request a
+    particukar .bob file please out the filename in the url path.
```

Comment:
There's a couple of typos here, and I can't tell what word `please` is supposed to be.